### PR TITLE
Docs: added Bezier example

### PR DESCRIPTION
**Pull request by kolibril13**
Some more documentation for the CubicBezier example

EDIT: example can be found here: https://manimce--1021.org.readthedocs.build/en/1021/reference/manim.mobject.geometry.CubicBezier.html?highlight=bezier#manim.mobject.geometry.CubicBezier

## Acknowledgements
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

File: manim/mobject/geometry.py [hunk 1]
```diff
@@ -912,6 +912,29 @@ def __init__(self, *args, **kwargs):
...
+    """
+    Example
+    -------
+
+    .. manim:: BezierSplineExample
+        :save_last_frame:
+
+        class BezierSplineExample(Scene):
+            def construct(self):
+                np.random.seed(42)
```
Comment (behackl) [inline manim/mobject/geometry.py:924]: There is no randomness in this example, is there?

```suggestion
```

File: manim/mobject/geometry.py [hunk 2]
```diff
@@ -912,6 +912,29 @@ def __init__(self, *args, **kwargs):
...
+                p1 = np.array([-3, 1, 0])
+                p1b = p1 + [1, 0, 0]
+                d1 = Dot(point=p1).set_color(BLUE)
+                l1 = Line(p1, p1b)
+                p2 = np.array([3, -1, 0])
+                p2b = p2 - [1, 0, 0]
+                d2 = Dot(point=p2).set_color(RED)
+                l2 = Line(p2, p2b)
+                bezier=CubicBezier([p1b, p1b + 3 * RIGHT, p2b - 3 * RIGHT, p2b])
+                self.add(l1, d1,l2, d2,bezier)
```
Comment (behackl) [inline manim/mobject/geometry.py:934]:
```suggestion
                self.add(l1, d1, l2, d2, bezier)
```

**Review by behackl — APPROVED** [review]
LGTM, added two suggestions.

File: manim/mobject/geometry.py (hunk 1 above)
Comment (kolibril13) [reply manim/mobject/geometry.py:924]: yep, that was only used in a previous version of this script

File: manim/mobject/geometry.py (hunk 2 above)
Comment (kolibril13) [reply manim/mobject/geometry.py:933]: thanks for noting this.

File: manim/mobject/geometry.py [hunk 3]
```diff
@@ -912,6 +912,28 @@ def __init__(self, *args, **kwargs):
...
+                p1 = np.array([-3, 1, 0])
+                p1b = p1 + [1, 0, 0]
+                d1 = Dot(point=p1).set_color(BLUE)
+                l1 = Line(p1, p1b)
+                p2 = np.array([3, -1, 0])
+                p2b = p2 - [1, 0, 0]
+                d2 = Dot(point=p2).set_color(RED)
+                l2 = Line(p2, p2b)
+                bezier=CubicBezier([p1b, p1b + 3 * RIGHT, p2b - 3 * RIGHT, p2b])
+                self.add(l1, d1, l2, d2, bezier)
```
Comment (jsonvillanueva) [inline manim/mobject/geometry.py:933]: Ran black formatting against the code:
```suggestion
        class BezierSplineExample(Scene):
            def construct(self):
                p1 = np.array([-3, 1, 0])
                p1b = p1 + [1, 0, 0]
                d1 = Dot(point=p1).set_color(BLUE)
                l1 = Line(p1, p1b)
                p2 = np.array([3, -1, 0])
                p2b = p2 - [1, 0, 0]
                d2 = Dot(point=p2).set_color(RED)
                l2 = Line(p2, p2b)
                bezier = CubicBezier([p1b, p1b + 3 * RIGHT, p2b - 3 * RIGHT, p2b])
                self.add(l1, d1, l2, d2, bezier)
```

**Review by jsonvillanueva — CHANGES_REQUESTED** [review]
I'm not sure if we should run black against documentation, but just a code suggestion

**Review by jsonvillanueva — APPROVED** [review]

**Comment by kolibril13** [comment]
> I'm not sure if we should run black against documentation, but just a code suggestion

I actually ran black on this file, however, it is not recognized because it is in the commented block.
Of course, one could copy every example to a separate fil, then run black and copy it back, but that seems to be a bit overkill for me.

**Comment by jsonvillanueva** [comment]
Yeah, I had to manually copy/paste the example from the commented block to a python file, run black there, then copy/paste it back/ indent it for the suggestion. I agree, very overkill to do manually. Would be cool for a future PR to format all existing examples, or figure a way to auto run black formatting on commented blocks.

**Pull request merged**